usaspending_api/references/migrations/0047_auto_20200621_1848.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 2.2.10 on 2020-06-21 18:48
```

Comment:
Are two separate migration files necessary for the data change operations or can they be combined?

Reply:
no reason; updated